pom.xml

```
@@ -9,16 +9,19 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
     <!-- set the version for Camunda Platform here -->
-    <version.camunda>7.19.0</version.camunda>
-    <version.junit>5.9.2</version.junit>
+    <version.camunda>7.20.0-alpha6</version.camunda>
```

🔧 I'll adjust this as soon as the PR CI is successful using the alpha.
```suggestion
    <version.camunda>7.20.0</version.camunda>
```